Makefile

```diff
@@ -18,7 +18,7 @@ GO_LDFLAGS += -s -w
 
 ARTIFACTS := _out
 
-TOOLS ?= ghcr.io/talos-systems/tools:v0.9.0
```

Comment:
we should bump TOOLS/PKGS to v1.0.0 while at it

Reply:
ok, let's do it